crates/test_utils/wasm/wasm_workspace/link/src/coordinator.rs

```diff
@@ -81,110 +81,145 @@ fn delete_link(create_link_hash: ActionHash) -> ExternResult<ActionHash> {
 #[hdk_extern]
 fn get_links(_: ()) -> ExternResult<Vec<Link>> {
     // Include just `SomeLinks`
-    hdk::prelude::get_links(GetLinksInputBuilder::try_new(base()?, LinkTypes::SomeLinks)?.build())?;
+    hdk::prelude::get_links(
+        LinkQuery::try_new(base()?, LinkTypes::SomeLinks)?,
+        GetStrategy::default(),
+    )?;
     // Include all links from within this zome.
-    hdk::prelude::get_links(GetLinksInputBuilder::try_new(base()?, ..)?.build())?;
+    hdk::prelude::get_links(
+        LinkQuery::new(base()?, (..).try_into_filter()?),
+        GetStrategy::default(),
+    )?;
     // Include types in this vec.
     hdk::prelude::get_links(
-        GetLinksInputBuilder::try_new(
+        LinkQuery::new(
             base()?,
-            vec![LinkTypes::SomeLinks, LinkTypes::SomeOtherLinks],
-        )?
-        .build(),
+            vec![LinkTypes::SomeLinks, LinkTypes::SomeOtherLinks].try_into_filter()?,
+        ),
+        GetStrategy::default(),
     )?;
     // Include types in this array.
     hdk::prelude::get_links(
-        GetLinksInputBuilder::try_new(base()?, [LinkTypes::SomeLinks, LinkTypes::SomeOtherLinks])?
-            .build(),
+        LinkQuery::new(
+            base()?,
+            [LinkTypes::SomeLinks, LinkTypes::SomeOtherLinks].try_into_filter()?,
```

Comment:
Possible to use `LinkQuery::try_new` here instead so you don't have to do this conversion?

Reply:
Unfortunately we can't.

The problem is that we've got a trait LinkFilterExt which is implemented for vec, range and slices, which has try_into_filter, but it's part of hdi and it does also this:

```rust
impl LinkTypeFilterExt for core::ops::RangeFull {
    fn try_into_filter(self) -> Result<LinkTypeFilter, WasmError> {
        let out = zome_info()?.zome_types.links.dependencies().collect();
        Ok(LinkTypeFilter::Dependencies(out))
    }
}
```

so I don't think it's possible to have `LinkFilterExt` as a dependency of `holochain_zome_types` (where `LinkQuery` is implemented). Previously the `GetLinksInputBuilder` used `impl LinkFilterExt` but it's located in `hdk`.

Because of that we first have to convert `impl LinkTypeFilterExt` to `LinkFilter` and then call `new`